app/src/main/java/comcmput301f16t01/github/carrier/UserProfileActivity.java

```diff
@@ -1,8 +1,111 @@
 package comcmput301f16t01.github.carrier;
 
+import android.os.Bundle;
+import android.provider.ContactsContract;
+import android.support.v7.app.AppCompatActivity;
+import android.view.View;
+import android.widget.EditText;
+import android.widget.ImageButton;
+import android.widget.TextView;
+
 /**
  * Created by kiete on 10/18/2016.
  */
 
-public class UserProfileActivity {
+public class UserProfileActivity extends AppCompatActivity {
+    @Override
+    protected void onCreate(Bundle savedInstanceState) {
+        super.onCreate(savedInstanceState);
+        setContentView(R.layout.activity_user_profile);
+        // Get the TextViews for the information that is going to be shown.
+        EditText userName = (EditText) findViewById(R.id.NameEditText);
+        EditText emailAddress = (EditText) findViewById(R.id.EmailEditText);
+        EditText phoneNumber = (EditText) findViewById(R.id.PhoneEditText);
```

Comment:
The comment mentions that these are TextViews, yet they are an EditText object. Please make sure that they cannot be edited! (only viewed) :)